modules/connectors/dpd_group/karrio/mappers/dpd_group/proxy.py

```diff
@@ -0,0 +1,56 @@
+"""Karrio DPD Group client proxy."""
+
+import karrio.lib as lib
+import karrio.api.proxy as proxy
+import karrio.mappers.dpd_group.settings as provider_settings
+import karrio.universal.mappers.rating_proxy as rating_proxy
+
+
+class Proxy(rating_proxy.RatingMixinProxy, proxy.Proxy):
+    settings: provider_settings.Settings
+
+    def get_rates(self, request: lib.Serializable) -> lib.Deserializable[str]:
+        return super().get_rates(request)
+
+    def _get_auth_headers(self) -> dict:
+        """Build authentication headers for DPD META-API."""
+        # access_token property returns the token string directly
```

Comment:
Check https://github.com/karrioapi/karrio/blob/904e5a8dfd8283a0c93948b99d57c7bdf385f859/modules/connectors/teleship/karrio/mappers/teleship/proxy.py#L15-L56 for the new standardized way of handling auth token. with caching, thread safety and such.

here is how other methods call it 

https://github.com/karrioapi/karrio/blob/904e5a8dfd8283a0c93948b99d57c7bdf385f859/modules/connectors/teleship/karrio/mappers/teleship/proxy.py#L59